packages/boot/README.md

```diff
@@ -0,0 +1,90 @@
+# @loopback/boot
+
+A collection of Booters for LoopBack Applications
+
+# Overview
+
+A Booter is a Class that can be bound to an Application and is called
+to perform a task before the Application is started. A Booter may have multiple
+phases to complete its task.
+
+An example task of a Booter may be to discover and bind all artifacts of a
+given type.
+
+BootComponent is responsible for handling Booter artifacts and running the
+phases. It must be added to an `Application` to use `await app.booter()`
+or `await app.boot()`.
```

Comment:
?

Reply:
I'll update the statement, but what I'm trying to say is you must have a `BootComponent` or equivalent in your app to use `app.boot()` or `app.booter()` since those are sugar methods for `BootComponent.boot()` / `BootComponent.booter()`.